tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -86,7 +86,8 @@ def test_check_sfputil_eeprom(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
             assert parsed_eeprom[intf] == "SFP EEPROM detected"
 
 
-def test_check_sfputil_reset(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, conn_graph_facts, tbinfo, xcvr_skip_list, shutdown_ebgp):
+def test_check_sfputil_reset(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, conn_graph_facts,
```

Comment:
@AharonMalkin can you come up with a better solution or have the optics installed on all ports? Adding vendors specific parameters in common test does not look good. What about other platforms? Add one more parameter to the test?